admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -1692,15 +1707,22 @@ var priceCalculation = (function() {
         priceCalculation.taxExclude();
       });
 
-      /** combinations : update TTC price field on change */
+      /** combinations : update TTC price field on HT change */
       $(document).on('keyup', '.combination-form .attribute_priceTE', function() {
         priceCalculation.impactTaxInclude($(this));
         priceCalculation.impactFinalPrice($(this));
       });
-      /** combinations : update HT price field on change */
+      /** combinations : update HT price field on TTC change */
```

Comment:
TTC is French if I am not wrong?

Reply:
Yep, but I used the same wording that was already present 🤷‍♂️